docs/pages/develop/project-structure.mdx

```diff
@@ -9,7 +9,7 @@ When you create a new Expo project, a few files and folders are provided by defa
 
 The **App.js** file is the default screen of your project. It is the root file that you'll load after running the development server with `npx expo start`. You can edit this file to see the project update instantly. Generally, this file will contain root-level React Contexts and navigation.
 
-> [Expo Router](/routing/introduction/) is now available and supports file-based navigation in projects. With Expo Router, the route file will become **index.js** and all the screens in the project will be in the **app** folder.
+> [Expo Router](/router/introduction/) is now available and supports file-based navigation in projects. With Expo Router, the route file will become **index.js** and all the screens in the project will be in the **app** folder.
```

Comment:
I know this was already there, but I'm not sure what "the route file" is here. Is it the initial screen?